.github/workflows/run_pr_tests.yml

```diff
@@ -194,3 +194,33 @@ jobs:
         hal_refsi_soc: G1
         hal_refsi_thread_mode: WG
         debug_support: ON
+
+  # Based on: mr-ubuntu-gcc-x86_64-riscv-fp16-cl3-0
+  run-ubuntu-gcc-x86_64-riscv-fp16-cl3-0:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 60
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: setup-ubuntu
+      uses: ./.github/actions/setup_build
+      with:
+        llvm_version: '19'
+        llvm_build_type: RelAssert
+    - run: echo WORKSPACE is $GITHUB_WORKSPACE && echo PWD is `pwd` && ls -al
+    - name: build ock
+      uses: ./.github/actions/do_build_ock
+      with:
+        build_targets: check-ock
+        mux_targets_enable: riscv
+        external_compiler_dirs: ${{ github.workspace }}/examples/refsi/refsi_m1/compiler/refsi_m1
```

Comment:
This is wrong, since we are using G1.

Reply:
Gitlab legacy setting - will change "m1" to "g1_wi"

Reply:
no, that's also wrong, it's no wi, just use the default.

Reply:
Done